wallet/test/Test/Pos/Wallet/Arbitrary/Web/ClientTypes.hs

```diff
@@ -17,7 +17,7 @@ import           Pos.Wallet.Web.ClientTypes.Types (CHash (..), CId (..),
 import           Pos.Wallet.Web.State (WAddressMeta (..))
 
 import           Test.Pos.Core.Arbitrary ()
-import           Universum
+import           Universum ()
```

Comment:
I do not think it is good change - `(<$>)`, `pure` and `(.)` used below are not in scope then.